### PR TITLE
Start docs for installation, use, and how to write a spider. Towards …

### PR DESCRIPTION
**Pull request by rhiaro**
…#280

### CODE REVIEW

File: docs/writing-scrapers.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+The Scrapy framework determines what happens to a crawled item through its 'pipeline', and lots of things happen in the background (like the actual sending of requests, and automatic passing things through different stages in the pipeline).
+
+Scrapy schedules HTTP requests with its own crawler engine, and they are not guaranteed (or likely) to be executed in a predictable order.
+
+`Scrapy documentation <https://docs.scrapy.org/en/latest/>`_.
+
+About OCDS Kingfisher
+~~~~~~~~~~~~~~~~~~~~~
+
+The OCDS data sources are for the most part JSON APIs which output valid OCDS data, either as ``release_package``, ``record_package``, ``release``, ``record`` and a few other types, using various different structures or endpoints of the publishers' own design.
```
Comment (yolile) [inline docs/writing-scrapers.rst]: maybe just "OCDS data" without "valid".  The "valid" word is kind of banned

File: docs/writing-scrapers.rst [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Our spiders need to:
+
+* Know the URL(s) for a particular API from which all data can be found.
```
Comment (yolile) [inline docs/writing-scrapers.rst]: maybe "source" instead of API, as the URL can be a file, etc too

File: docs/writing-scrapers.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+* Tell the difference between endpoints which return lists of URLS
+   * And then follow these URLS
+* and endpoints which return OCDS data
+   * And then download this data.
+* and endpoints which return both useful OCDS data and more URLs to fetch
+* and endpoints which return other things.
+
+This tends to involve prior knowledge of the API you're writing a spider for (you have to go look at its responses yourself to see what they are), and maybe some JSON parsing of the responses.
```
Comment (yolile) [inline docs/writing-scrapers.rst]: same, maybe "source"

**Comment by odscjames** [comment]
Closing this pull request as it now has to be against master branch.

File: docs/writing-scrapers.rst (hunk 1 above)
Comment (odscjames) [reply docs/writing-scrapers.rst]: I removed the V word in some of my edits

Comment (odscjames) [reply docs/writing-scrapers.rst]: Oh, in a different place tho - I will keep an eye on that file too

**Comment by jpmckinney** [comment]
I changed the base branch – seems like it's in master, so deleting branch.

**Comment by odscjames** [comment]
Yes, we did this in another pull request (I didn’t know you could just get github to change the target, will do that in future)

**Pull request closed without merging**